deepchecks/vision/utils/detection_precision_recall.py

```diff
@@ -54,7 +54,7 @@ def compute(self):
                 **self._compute_ap_recall(ev["scores"], ev["matched"], ev["NP"])
             }
         if self.return_ap_only:
-            res = torch.tensor([res[k]['AP'] for k in sorted(res.keys())])
+            res = torch.tensor([res[k]["AP"] for k in sorted(res.keys())])
```

Comment:
why?

Reply:
pylint told me

Reply:
thought our accepted format was ''. Perhaps it told you that because other places in this doc are "". In that case, need to change all to '' (but maybe wait for @ItayGabbay recent pylint commits)

Reply:
I changed it to "" in this particular file.